### PR TITLE
DOC-1433: Add Cloud API examples for managing a subscription in API "Create and edit subscriptions" page

### PR DESCRIPTION
**Pull request by cmilesb**
Staging link: https://docs.redis.com/staging/jira-doc-1433/rc/api/examples/manage-subscriptions/

### CODE REVIEW

File: content/rc/api/examples/manage-subscriptions.md [hunk 1]
```diff
@@ -11,11 +11,11 @@ aliases: /rv/api/how-to/create-and-manage-subscriptions/
          /rc/api/examples/manage-subscriptions.md
 ---
 
-You can use the Redis Enterprise Cloud REST API to create and manage a subscription.
+The Redis Enterprise Cloud REST API allows you to create and manage a subscription.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:14]: I would avoid "allows you to". You could either start with the verb: 
Create and manage your subscription with the Redis Enterprise Cloud REST API. 
or use "lets" 
The Redis Enterprise Cloud REST API lets you create and manage your subscription.

File: content/rc/api/examples/manage-subscriptions.md [hunk 2]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
 }
 ```
 
-To use the sample JSON document in your own account, you must modify these parameters:
+To use the sample JSON document in your own account, modify these parameters:
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:47]: I would start this with the action: 
Modify the [following/these] parameters in the sample JSON document to create a subscription on your own account.

File: content/rc/api/examples/manage-subscriptions.md [hunk 3]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
 
-- **`paymentMethodId`** - Specify a payment method that is defined for your account.
+- **`paymentMethodId`** - Specify a payment method that is connected to your account.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:49]: I would say either: 
...connected to your account. 
or
...for your account.
"that is connected to" is too many words

File: content/rc/api/examples/manage-subscriptions.md [hunk 4]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
 
-    You can lookup the payment method identifier using the `GET /payment-methods` API operation.
+    Use `GET /payment-methods` to find a payment method ID.
 
-    If you subscribed to Redis Enterprise Cloud through the GCP Marketplace, you do not need to pass this field in your API requests.
+    You do not need to pass this field in your API request if you subscribed to Redis Enterprise Cloud through GCP Marketplace.
 
-- **`cloudAccountId`** - Specify a cloud account that is defined for your account.
+- **`cloudAccountId`** - Set a cloud account ID that is connected to your account.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:55]: "that is connected to" -> same comment as above

File: content/rc/api/examples/manage-subscriptions.md [hunk 5]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+The request JSON body contains two primary segments: subscription specification and databases specification. When you create a subscription, you must specify one or more databases in the "`databases`" array of the above JSON file.
+
+You can [copy-and-paste]({{< relref  "/rc/api/get-started/use-rest-api.md#swagger-user-interface" >}}) the contents of the JSON file into the `POST /subscriptions` operation in the [Swagger UI](https://api.redislabs.com/v1/swagger-ui.html).
 
 {{< note >}}
 The Swagger UI generates default JSON examples for `POST` and `PUT` operations. You can reference these examples and modify them to fit your specific needs and account settings. The examples will fail if used as-is.
 {{< /note >}}
 
-The POST response is a JSON document that contains the `taskId`. You can use `GET /v1/tasks/<taskId>` to track the creation of your subscription.
+The response body contains the `taskId` of the task that is creating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:69]:
```suggestion
The response body contains the `taskId` for the task creating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 6]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+```
+
+You can only change the following settings with this endpoint:
+- **`name`** - Specify a new name for your subscription.
+
+- **`paymentMethodId`** - Specify a different payment method that is defined for your account.
+
+    You can look up a payment method identifier using the `GET /payment-methods` API operation.
+
+The response body contains the `taskId` of the task that is updating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:90]:
```suggestion
The response body contains the `taskId` for the task updating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 7]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+{
+    "name": "new-subscription-name",
+    "paymentMethodId": <payment_id>
+}
+```
+
+You can only change the following settings with this endpoint:
+- **`name`** - Specify a new name for your subscription.
+
+- **`paymentMethodId`** - Specify a different payment method that is defined for your account.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:86]:
```suggestion
- **`paymentMethodId`** - Specify a different payment method for your account.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 8]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+The response body contains the `taskId` of the task that is updating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
+
+## Delete a subscription
+
+Use `DELETE /v1/subscriptions/<subscriptionId>` to delete a subscription.
+
+```sh
+DELETE "https://[host]/v1/subscriptions/<subscriptionId>"
+```
+The response body contains the `taskId` of the task that is deleting the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:99]:
```suggestion
The response body contains the `taskId` for the task deleting the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 9]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+To use the sample JSON document in your own account, modify these parameters:
 
-- **`paymentMethodId`** - Specify a payment method that is defined for your account.
+- **`paymentMethodId`** - Specify a payment method that is connected to your account.
 
-    You can lookup the payment method identifier using the `GET /payment-methods` API operation.
+    Use `GET /payment-methods` to find a payment method ID.
 
-    If you subscribed to Redis Enterprise Cloud through the GCP Marketplace, you do not need to pass this field in your API requests.
+    You do not need to pass this field in your API request if you subscribed to Redis Enterprise Cloud through GCP Marketplace.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:53]:
```suggestion
    You don't need to pass this field in your API request if you subscribed through GCP Marketplace.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 10]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
 
-- **`cloudAccountId`** - Specify a cloud account that is defined for your account.
+- **`cloudAccountId`** - Set a cloud account ID that is connected to your account.
 
-    You can look up cloud account identifiers using the `GET /cloud-accounts` API operation or use `"cloudAccountId": 1` to use internal resources.
+    You can look up cloud account IDs using `GET /cloud-accounts`. To use internal resources, set it to `"cloudAccountId": 1`.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:57]:
```suggestion
    To list cloud account IDs, use `GET /cloud-accounts`. 
    To use internal resources, set `cloudAccountID` to`"cloudAccountId": 1`.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 11]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
 
-    If you subscribed to Redis Enterprise Cloud through the GCP Marketplace, use the value `1` for this field.
+    If you subscribed to Redis Enterprise Cloud through GCP Marketplace, use `1` for this field.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:59]:
```suggestion
    If you subscribed through GCP Marketplace, use `1` for this field.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 12]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
 
-- The JSON document contains two primary segments: subscription specification and databases specification.
-- When you create a subscription, you must specify one or more databases in the "`databases`" array of the above JSON file.
-- You can [copy-and-paste]({{< relref  "/rc/api/get-started/use-rest-api.md#swagger-user-interface" >}}) the contents of the JSON file into the `POST /subscriptions` operation in the [Swagger UI](https://api.redislabs.com/v1/swagger-ui.html).
+The request JSON body contains two primary segments: subscription specification and databases specification. When you create a subscription, you must specify one or more databases in the "`databases`" array of the above JSON file.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:61]:
```suggestion
The request JSON body contains two segments: subscription specification and databases specification. When you create a subscription, you must specify one or more databases in the "`databases`" array.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 13]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+    "paymentMethodId": <payment_id>
+}
+```
+
+You can only change the following settings with this endpoint:
+- **`name`** - Specify a new name for your subscription.
+
+- **`paymentMethodId`** - Specify a different payment method that is defined for your account.
+
+    You can look up a payment method identifier using the `GET /payment-methods` API operation.
```
Comment (kaitlynmichael) [inline content/rc/api/examples/manage-subscriptions.md:88]: To find the IDs for payment methods connected to your account, use he `GET /payment-methods` API operation.

**Review by kaitlynmichael — CHANGES_REQUESTED** [review]
Left some suggestions to tighten up the wording. Let me know if you have any questions :)

File: content/rc/api/examples/manage-subscriptions.md [hunk 14]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
...
+The response body contains the `taskId` for the task updating the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
+
+## Delete a subscription
+
+Use `DELETE /v1/subscriptions/<subscriptionId>` to delete a subscription.
+
+```sh
+DELETE "https://[host]/v1/subscriptions/<subscriptionId>"
+```
+The response body contains the `taskId` for the task deleting the subscription. You can use `GET /v1/tasks/<taskId>` to track the status of this task.
```
Comment (rrelledge) [inline content/rc/api/examples/manage-subscriptions.md:99]:
```suggestion
The response body contains the `taskId` for the task that deletes the subscription. You can use `GET /v1/tasks/<taskId>` to track the task's status.
```

File: content/rc/api/examples/manage-subscriptions.md [hunk 15]
```diff
@@ -44,26 +44,56 @@ POST "https://[host]/v1/subscriptions"
 }
 ```
 
-To use the sample JSON document in your own account, you must modify these parameters:
+Modify the following parameters in the sample JSON document to create a subscription on your own account.
```
Comment (rrelledge) [inline content/rc/api/examples/manage-subscriptions.md:47]:
```suggestion
Modify the following parameters in the sample JSON document to create a subscription on your own account:
```

**Review by rrelledge — APPROVED** [review]
Looks good to me. I left a few suggestions for you to consider. Also, should the examples show [host]:[port]?

**Pull request merged**